File: selection sort.py
def ssort(lst):
    l1 = len(lst);
    for i in range(0,l1):
        mini = 100000;
        sub = 0;
        for j in range(i,l1):
            if lst[j] < mini:
                sub = j;
                mini = lst[j];
        t = lst[i];
        lst[i] = lst[sub];
        lst[sub] = t;
    return;

lst = [30,21,5,7,1];
ssort(lst);
print (lst);

File: selection sort.cpp
#include<cstdio>
#include<iostream>
using namespace std;
double num[10000];
int ssort(int l)
{
	int sub=0;
	for(int i=0;i<l;i++)
	{
		double min=100000;
		for(int j=i;j<l;j++)
		{
			if(num[j]<min)
			{
				min=num[j];
				sub=j;
			}
		}
		int c=num[i];
		num[i]=num[sub];
		num[sub]=c;
	}
	return 0;
}
int main()
{
	int l;
	cin>>l;
	for(int i=0;i<l;i++)
		cin>>num[i];
	ssort(l);
	for(int i=0;i<l;i++)
		cout<<num[i]<<" ";
	return 0;
}
